R/html_resources.R

```diff
@@ -189,8 +197,8 @@ discover_rmd_resources <- function(rmd_file, encoding,
   rmd_content <- read_lines_utf8(rmd_file, encoding)
   writeLines(rmd_content, md_file, useBytes = TRUE)
 
-  # create a vector of temporary files; anything in here will be cleaned up on
-  # exit
+  # create a vector of temporary files; anything in here
+  # will be cleaned up on exit
```

Comment:
Looks like you use 55 as the width? Mine is 80 (the default in RStudio).

Reply:
I mostly try to make the comments and examples a certain (shorter) width and try to keep everything else within 80.